### PR TITLE
Further east patch 2

### PR DESCRIPTION
**Pull request by further-east**
Should be fixed now, sorry for the error

### CODE REVIEW

**Review by mbg — CHANGES_REQUESTED** [review]
The formatting is still broken. Also, please don't open new PRs for changes. Any changes you push to your branch are automatically reflected in the corresponding PR.

**Comment by further-east** [comment]
Terribly sorry for the inconvenience, is the formatting correct now?

File: config/groups.yaml [hunk 1]
```diff
@@ -740,6 +740,11 @@
     - "2833207"
     - "2637174"
   id: 190178
+ - name: "The prototype"
+  members:
```
Comment (mbg) [inline config/groups.yaml:744]:
```suggestion
- name: "The prototype"
  members:
```

**Review by mbg — CHANGES_REQUESTED** [review]
No, see my suggestion for how to fix it.

**Review by mbg — APPROVED** [review]

**Pull request merged**